search/indexing_api.py

```diff
@@ -193,13 +193,12 @@ def serialize_program_enrolled_user(program_enrollment):
 def filter_current_work(profile):
     """
     Remove work_history objects that are not current
+    Args:
+        profile: serialized profile
```

Comment:
Can you change `profile` here and below to be `Profile`, to exactly match the profile type?